Part_Three/PartThree_Shared_Functionality.py

```diff
@@ -0,0 +1,45 @@
+
+EMPTY_STRING = ''
+NOT_VALID_ROW_MSG = 'There is not a valid row in the file'
+
+
+# The object for both parts
+class SharedObject(object):

```

Comment:
Names of classes should reflect what the class is, and not a property of the class.
from this name, I do understand that this class is shared between other parts of the code (which is basically true for all the classes, no? we using classes to encapsulate shared functionality). But I can't understand this class is? what is the shared logic?
Try to think about it while naming your classes.